packages/next/src/lib/metadata/generate/basic.tsx

```diff
@@ -157,6 +157,22 @@ export function FacebookMeta({
   ])
 }
 
+export function PinterestMeta({
```

Comment:
The render part of this component is missing, yu need to render it in `packages/next/src/lib/metadata/metadata.tsx` and please also add an e2e test case for it in metadata.test.ts